app/views/unattended/provisioning_templates/snippet/kickstart_kernel_options.erb

```diff
@@ -29,7 +29,12 @@ snippet: true
 
   # tell Anaconda what to pass off to kickstart server
   # both current and legacy syntax provided
-  options.push("kssendmac", "ks.sendmac", "inst.ks.sendmac")
+
+  if (is_fedora && os_major >= 34) || (rhel_compatible && os_major >= 9)
+    options.push("inst.ks.sendmac")
+  else
+    options.push("kssendmac", "ks.sendmac", "inst.ks.sendmac")  
+  end
```

Comment:
Thank you, I am gonna ask for more work but if you don't feel comfortable just say a word and we can merge this as-is.

I think we can move the "ks/inst.ks" option in here, since kickstart_kernel_options snippet is used pretty much in all kickstart types, that would be a nice place to have it. What you think?

Reply:
I think it seems like a good way to DRY, which is usually a plus.  I believe I moved the change into the kickstart_kernel_options snippet.